js/index.js

```diff
@@ -1,42 +1,90 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
+  //console.log('Calculating subtotal, yey!');
```

Comment:
I would recommend removing any code/comment which no longer serves a purpose, it'll keep your code minimal and easier to read through